package.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "debug": "^4.3.4",
     "lodash": "^4.17.15",
-    "mongodb": "^4.9.0"
+    "mongodb": "^5.7.0"
```

Comment:
@shafeeqd959 / @cs-raj  Have we conducted validation for the breaking changes introduced in versions v5 and above?

https://github.com/mongodb/node-mongodb-native/releases/tag/v5.0.0
https://github.com/mongodb/node-mongodb-native/compare/v4.9.0...v5.7.0

cc @netrajpatel

Reply:
@antonyagustine  I tested it. it worked fine
cc: @shafeeqd959 @netrajpatel